src/components/RichTextEditor/index.scss

```
@@ -58,6 +58,7 @@
   }
 
   .toastui-editor-defaultUI {
+    z-index: 0;
```

```suggestion
    // z-index of 0 here prevents a few strange interactions with other UI elements not being clickable when they overlap the editor
    z-index: 0;
```

Just a quick suggestion on a comment for this for posterity!